### ISSUE TITLE
Need more Secanrios - not just SSW's Free Lunch Email

### ISSUE THREAD

**Issue by brendan-nobadthing** [wicksipedia/PointBank#12]
First, I think that there has been some fantastic work designing and documenting the idea for this project. 

My concern is that the problem with most SSW internal projects is that they end up solving "SSW-specific" problems and end up being less useful for anyone else.

Before building we should  investigate at least one other sceanario that a non-SSW customer has and design a workflow for that. The risk is we will end up building an automated "SSW free lunch system" that noone else wants.

**Comment by matthewsampiasssw** [comment]
I agree - I see a few opportunities within the presentation for potential other products that we could produce from this including:
1. a middleware integration product for productivity apps
2. a gamification tool that could be applied to jobs with repetitive tasks such as trade jobs, testing, etc. 
3. a mobile app that could be used to track work not done on a laptop or computer

We would probably want to verify these scenarios and do some market research with a test client who could be the first user.

**Comment by calumjs** [comment]
Agreed, the free lunch is just an example and we want it to be as generic as possible, but an extended list of use cases would be cool.
If you can think of some, feel free to add them as issues e.g.
#16 #8 #9 #10

**Comment by JerryLuo5799** [comment]
I agree - we need to consider more usage scenarios. It should evolve into a general-purpose middleware
